src/app/view/web/WebChessView.ts

```diff
@@ -21,14 +21,33 @@ export class WebChessView implements ChessBoardView {
     this.viewEventBus.listenOn(eventType, reaction);
   }
 
-  showSelectedPiece(position: { x: number; y: number }): void {}
+  showSelectedPiece(id: string): void {
+    if (this.parent.querySelector(`#${id}`)?.hasChildNodes()) this.parent.querySelector(`#${id}`)?.classList.add('square--selected');
```

Comment:
Co prawda sam jeszcze nie cierpiałeś z tego powodu, ale polecałbym jednak oduczyć się robienia ifów bez {} :) Kiedyś to się zemści. Jak ktoś w rekrutacji będzie oglądał Twój kod - też lepiej żeby były.